src/System.Net.Requests/ref/System.Net.Requests.cs

```diff
@@ -406,21 +406,4 @@ public class HttpRequestCachePolicy : RequestCachePolicy
         public System.TimeSpan MaxStale { get { throw null; } }
         public override string ToString() { throw null; }
     }
-    public enum RequestCacheLevel
```

Comment:
I assume these are new correct? If they have shipped then we need to leave behind a type-forward.